### PR TITLE
Updates heroku docs + adds link

### PR DESCRIPTION
**Pull request by tiffine-koch**

### CODE REVIEW

**Comment by caoyangs** [comment]
a lot of empty lines?

**Comment by tiffine-koch** [comment]
Yes, I'm not sure how that happened, only edits I had were in deploy.markdown. Will look into it and resubmit.

**Comment by tiffine-koch** [comment]
I deleted the html files in my forked repo. The are not in the electrode-io develop branch.
I will ask @melophonic to review as well; I used the diffing tool to double check that I have the same files in my new fork, but I will make sure before merging.

File: css/electrode.css [hunk 1]
```diff
@@ -0,0 +1 @@
...-list-item{margin-top:60px}.hll{background-color:#f7ebc6;border-left:5px solid #f7d87c;display:block;margin-left:-14px;margin-right:-14px;padding-left:9px}.highlight .javascript .err{background-color:transparent;color:inherit}.highlight{position:relative;margin-bottom:14px;padding:30px 14px 14px;border:none;border-radius:0;overflow:auto}.highlight pre{padding:0;margin-top:0;margin-bottom:0;background-color:transparent;border:0;float:left;min-width:100%}.highlight pre code{display:block;background:none;padding:0}.highlight pre .lineno{display:inline-block;width:22px;padding-right:5px;margin-right:10px;color:#bebec5;text-align:right}.highlight:after{position:absolute;top:0;right:0;left:0;padding:3px 7px;font-size:12px;font-weight:bold;color:#c2c0bc;background-color:#f1ede4;content:"Code"}.downloadCenter{text-align:center;margin-top:20px;margin-bottom:25px}.downloadSection:hover{text-decoration:none !important}@media screen and (max-width: 960px){.nav-main{position:static}.container{padding-top:0}}.post{margin-bottom:30px}.post img{max-width:100%}.pagination{margin-bottom:30px;width:100%;overflow:hidden}.pagination .next{float:right}.logo-blue{color:#F1BF6D}.logo-orange{color:#54a7de}
```
Comment (ananavati) [inline css/electrode.css]: this is the generated compiled css file, it shouldn't be checked in,  changes in `electrode.scss` should be checked-in

**Pull request merged**